docs/concepts/resource-manager.md

```diff
@@ -144,6 +144,13 @@ A `ManagedResource` has an optional `.spec.class` field that allows it to indica
 The `.controllers.resourceClass` field in the component configuration restricts the watch to `ManagedResource`s with the given `.spec.class`.
 A default class is assumed if no class is specified.
 
+The seed GRM watches resources with class `seed` in all namespaces.
+THe shoot GRM watches resources with the default class and only in it's designated namespace via `.sourceClientConnection.namespace`
```

Comment:
The reason i added it was because `.sourceClientConnection.namespace` is mentioned in this section and then never referenced again in the documentation. With the added sentences it contextualises why we have this option. What do you think about this new version?

> Note: The current implementation that handles `ManagedResource`s does it in the following way:
> 1. The seed Gardener Resource Manager watches resources with class `seed` in all namespaces.
> 2. The shoot Gardener Resource Manager watches resources with the default class and only in it's designated namespace via `.sourceClientConnection.namespace`